majasdarbs-05-2022/src/io/codelex/arithmetic/practice/number3.java

```diff
@@ -0,0 +1,31 @@
+package io.codelex.arithmetic.practice;
+
+import java.sql.Array;
+import java.util.Arrays;
+import java.util.stream.IntStream;
+public class number3 {
+
+    public static void main(String[] args) {
+        SumAverageRunningInt();
+    }
+    public static void SumAverageRunningInt() {
+
+        int start = 1;
+        int end = 100;
+
+        int[] arr = IntStream.rangeClosed(start,end).toArray();
```

Comment:
Es šo risinājumu pieņemu, bet man liekas, ka nav vērts tos ciparus glabāt masīvā - ja būtu tas range no 1 līdz 1 000 000, tad būtu muļķīgi viņus glabāt masīvā, ja varētu vienkārši ar for loop iet no 1 līdz beigām neturot tos atmiņā.